community-containers/nextcloud-exporter/nextcloud-exporter.json

```diff
@@ -0,0 +1,37 @@
+{
+    "aio_services_v1": [
+        {
+            "container_name": "nextcloud-aio-nextcloud-exporter",
+            "display_name": "Prometheus Nextcloud Exporter",
+            "documentation": "https://github.com/nextcloud/all-in-one/tree/main/community-containers/nextcloud-exporter",
+            "image": "ghcr.io/xperimental/nextcloud-exporter",
+            "image_tag": "0.8.0",
+            "internal_port": "9205",
+            "restart": "unless-stopped",
+            "depends_on": [
+                "nextcloud-aio-nextcloud"
+            ],
```

Comment:
This adjusts the startup order and I think it should not be needed. If you do this, it will be started after the nextcloud-aio-nextcloud and stopped before nextcloud-aio-nextcloud. Is this the intended behaviour?
```suggestion
```

Reply:
No true no need for that actually, the exporter will simply fail to fetch the data a few times. Which it does anyway until token authentication is configured.